### DISCUSSION TITLE
Adding More Entertainment Lore in Human/Adding a new alien races

### DISCUSSION THREAD

**Discussion by TheShadowOfHassen** [endless-sky/endless-sky#8202] [Ideas]
Sorry for the big title but its one thing that leads to the other and the second part is optional.

So A few years ago I played endless sky r and loved it and only with the recent steam update so my up to date-ness on the lore. But on replaying I had the feeling I had before that the entertainment edge of the endless sky world was a bit lacking. You have There Might Be Riots and mentions of other bands and a few mentions of watching soap operas and also someone talking about world building a story.

So I had an idea of wouldn't it be cool to add some jobs/missions/dialog that would fully flesh out this section of human space. It snowballed into a major plot arc with new alien species, Which are optional or can wait....

So first the entertainment lore is pretty shallow, and I thought I'd patch that up by adding ship and people talking about books/ movies. There are already hails talking about holo videos, If you want holos for the main video replacements that's fine, but personally I think that's a bit too star wars-y. Also, some jobs where you curry book manuscripts or take famous actors places to make that solid. Also book transports. There is a mention of a book in the game already, so there are them.

For missions, I was thinking of adding a few, and they would eventually join with the alien race eventually. There may be more missions made as I go on because I'm sort of a discovery writer.

My first idea is when you get a bit of money you help a indie movie director make their movie and become the producer. For the story there needs to be a mostly abandoned science station somewhere, and they use it for filming a horror movie. There'd be missions to bring people and actors  and cost you a bit of money (to help fund the movie). It's a nice low stakes story. There is a point where you think the station is actually haunted and that is important later.

My second idea follows a writer who gets way too famous. You meet him when he is a beginner ideally at the beginning of your time as a pilot, and he'd be a friend for you throughout the game. You'd get missions from him every so often and he'd send you notes after the major points in the story. He gets along a lot and learns about each alien race sometime after you do. (You can tell him but if you don't he'll find out himself) At the end (Once the free world story is done) He gets to famous and attracts the attention of a powerful pirate lady who also has alien tech. I don't know how I'd be able to justify having this be a hard mission, but it is and at the end  ship de-cloaks that is from an unknown alien race who help you fight of the pirates and then ask you for help.

So this brings in the aliens, two alien races in a major plot. This might need to be done in sections later, namely after the republic side of the free world is done. I'll write that instead of this alien campaign if no one else is working on it because I'm currently stuck in my game because I don't want to join the free worlds and I think it'd better to get that done before going to the feature.

 One thing that Endless sky doesn't have is a massive alien invasion story.  A big campaign where aliens need to work together against a large alien attack. Two alien races that are so far out that they haven't been interfered with by Draks Quarg or Pug.(Somewhere Far, West and slightly north of the Deep.) One of these aliens are in what the quarg said is its expansion phase and have stumbled upon some weapons the Quarg warned you about on your first contact. I don't know exactly why they are but they are trying to invade south. I'll work that out.

The other alien race is basically another wish for endless sky. I found that the large battles are just you flying around and hoping your ships were stronger than the enemies. There may be a bit more skill to it, but I haven't learned it, nothing has really taught me in game. This other group of aliens would be weaker in raw strength but know such strategies. Things sort of like what TImothy Zahn has Grand Admiral Thrawn do in his books. Using cloaking devices to mimic shooting patterns, use fighters as projectiles etc. I don't think those things can be done in endless sky, but some strategies could be implemented, and I'm open to suggestions on what.

Basically at the beginning this other set of aliens (Which are militaristic) ask you for help. (Somehow the writer person has met them) They give you an advanced jump drive for the 15 some non hyperspace lane jumps you have to make. You do some missions  but eventually the defending aliens would crumble, and the others would escape. I don't know what would happen next, but I'll figure something out.

Sorry for this post's length. I'll try to find names for the characters and Aliens. You all can tell me if you think this idea would fit what you think the game should go. Also I'm not the best at art and the new aliens will need ships so I might need some help there, but as you can see there is some time before it even gets to that. Other than art I can do a lot of the work myself.

**Comment by RisingLeaf** [comment]
So... lets sort this a bit, you got a whole lot of ideas and I will try to give my opinion on them.

So first, I like the idea of organizing a movie. It could be a fun side story and I would personally enjoy this. As one of the lead dev said, it would dig the hole(meaning the amount of plot) in human space deeper.

The second idea about the author is a bit more ambiguous in my opinion, I have no real opinion about it, but it sounds like a lot of work and I do not know how it would work with the lore.

Now to the ideas that I find really interesting, large scale invasion.
I have some question about this:

1. What tech levels are these aliens on?
2. Why are the Archons not cutting of these species from another?
3. What impact would the player have on the result of the invasion?
4. In ES we always try to not just black and white but have arguments for both sides, the current concept seems more black and white'ish

If I got your post right, you want to create these species? I am not sure because you said wish.
But if you plan on creating these species, I will tell you what we tell everyone:
It takes time, the most recent species addition (the gegno) has been in work for over a year. So if you plan on doing something for vanilla you need a lot of patience and you need to be open for changes on your ideas. On the other hand you can always create a plugin.

**Comment by Ishaname** [comment]
This sounds like a lot of work, but I feel like a lot of these sound like fun. If you aren't making a plugin yourself, I would be willing to help with this project. I need a little more experience with this sort of work, since I want this to be a full-time job, so I think this would be mutually beneficial. If this is of interest to you, feel free to contact me on Discord at Ishaname#4888 or email me at ishaname23@gmail.com.

**Comment by TheShadowOfHassen** [comment]
So I've done a lot more planning since my last post so I have a better grasp on the idea than I did.

Also I looked back and the stuff I originally posted looked pretty hard to read. I also revised a lot of it, so. I'll separate the parts.

So first I've gotten rid of the martial aliens because I had no clue where they went after the other aliens took over. I replaced them with another smaller faction.

Also the writer is being cut. There will be a writer in the human writing stuff and he'll probably get famous and you'll have to do something like take his books up to Hai but he won't be in the main plot.

Instead for a third sort of mission that you think is unesssary you take an expert in old earth languages: Like really an expert knowing norse old english latin etc. Actually statistically I doubt there's many people who can speak those languages now so it'd be rare to find someone in the future. He'd ask you for aliens because he wants to learn there languages but if you don't tell him about it nothing happens.

So now also because the story took way to time I cut it short right at the alien invasions. After the free worlds and probably after the new Hai reveal story I saw someone was making. Massive alien ships start taking planets in human space. It doesn't matter where. Thuese are the Norislor (There may need to be a special character over the first _o_ so its Nòrislor butI don't know if endless sky can handle that.)

I will say right now I think I have the black and white handled. I'll get to it.  Also I'm not 100% sure I have the lore memorized of endless sky so If I have something inconsistent or impossible let me know.

But i'm going to break to describe the Norislor


# Norislor

Norislor are a foot taller than you bulkier and have bluish gray skin. They wear armor. They are a war like species who conquers planets and fights for fun (Because there are species that are like that, Romans) But they are in human space for a reason.

They have planet destroying weapons and other terrible things. There tech level has to be a 4 so there on equal to Archon/ Drak or they'd get beaten up in a moment. They have jump drives.

These guys have massive ships with massive outfit and weapons space so I think their outfit balance will err on the side of really high quality but really really big.

#Now back to the story

There are two types of reactions in human galaxy. "We need to kill them," or, "We need to figure out what they are doing" Before the Norislor attacked they hailed the first planet and the audio was taken out. The player is messaged by the linguistics expert to go meet him because he's found out the language. They can do that or you can join the defense. This time the branch makes no difference. It ends basically the same way. (All the plots always branch but this time I'd like to show that the player cannot always change the fate of the universe)

If the player chooses to attack which is what the Quarg and maybe Archon want they gather forces from among the human and hai spaces (If that's what the Hai reveal outcome is) and then get slaughtered. It does halt the Norislor advance for some reason. You are out of options so the player goes see the language expert.

If they don't attack the fleet is destroyed right before you meet the language expert.

The language expert (Who needs a name) would reveal that according to the audio clip the Norislor are speaking in ancient norse. (There is a nordic twist to this story in a sort of explanation of norse mythology. I hope its alright) The messages demands the humans to tell them where there gods are. You show this to someone in power. I don't know who.

The next mission is the player taking the expert and a diplomat to talk to the aliens. The leader of the Norislor is Lord 
Bakrauf. He is not really helpful but gives the player more information:

He says that long ago they were fighting among themselves for their crown.(Nuclear fighting)  A weak son of there lord had the crown (Something's lost in translation of crown) A wormhole opened above there planet and ship poured in they hailed them and declared that they should stop fighting. The Norislor refused and attacked these ships. The ships fought back (And well) but they were defeated.  But at the last part of the battle one of the ships boarded the ship with the lord (with the crown) and stole it flying through the wormhole

The Norislor angry chased through the wormhole. Then through another one (Some sort of staging ground for the pug's "projects") and found earth, humans and their "gods" Pale skinned creatures (Pug) The Norislor attacked. They're ships were so thick that it covered the earth and sun and they were only stopped by the weapons that were sold to the gods. The Norislor were driven back through the wormholes without the crown and the wormholes shut behind them.

Now a break for explanation. The Norislor are supposed to be the jotun in old norse mythology. Frost giants basically and that battle was Ragnorok. The pug tried to stop their fighting but somehow misjudged the tier they needed to use on the Norislor. The people who sold the weapons I'll get to.

The crown is an important to the Norislor. Without it you basically have no hold on lordship. For the past 3,000 years they've been constantly fighting each other only stopping when someone strong takes lead and starts conquering, trying to find earth because thats where the pale ones who stole the crown were.

They make a deal saying that the Norislor will retreat from human space if they have the crown but if its not back in a year the Norislor will attack everything.

Back to the story...

This may also require the Wander/ pug part of the story done because the next thing the player has to do is find a pug. The pug is no help they said they gave the crown to the people they got the weapons too. The Dver. Also they say that the Dver are quite close to earth. They are in a system that no one else uses

The dver are basically dwarves from norse mythology and are a really old and advanced race without a planet. They live in a Dyson sphere cloaked around an uninhabited planet near earth. They know english. Also you can give the dvers certain peice of technology to "upgrade" if you give them raw materials (Astroids) It's a cool idea that encourages astroid mining and gives you upgrades.

Back to the story...

The player tries to convince the diver to give them the crown. The dvers agree but they want a planet. They don't have one. The ambassador agrees and they take the crown to the Norislor. Lord Bakrauf takes it but says in order for them to retreat there must be a battle of single combat. "For honor" he says. (Bakrauf has to do it to save face against the other Norislors)

The player has to disable the ship and then the Norislor leave.

That's a lot. Sorry but that's the idea.

@Ishaname; I would love help. Your plugin looks like you can do art!. Once the story is worked out I think I'll create a repository to develop the idea as a plugin for testing (If there's another better way to do this someone tell me) You can pull request to that.

**Comment by Saugia** [comment]
Hoo boy, you've got a lot here to present. I'll do my best in addressing what I think is important. 

Please be aware that this feedback is done if you're planning on adding things to *Vanilla* Endless Sky. If you are going to make this a plug-in, by all means go for it!


>So I had an idea of wouldn't it be cool to add some jobs/missions/dialog that would fully flesh out this section of human space. It snowballed into a major plot arc with new alien species, Which are optional or can wait....

Humans have enough plot links with other alien species, and especially with the current state of Hai Reveal, we don't want yet another side plot that snowballs into another arc, especially with another new alien species. Generally we limit each race with how many links they have to others, and the humans definitely don't need another brand new alien race interaction, especially one as major as you're suggesting.

>So first the entertainment lore is pretty shallow, and I thought I'd patch that up by adding ship and people talking about books/ movies. There are already hails talking about holo videos, If you want holos for the main video replacements that's fine, but personally I think that's a bit too star wars-y. Also, some jobs where you curry book manuscripts or take famous actors places to make that solid. Also book transports. There is a mention of a book in the game already, so there are them. 

This could be interesting with proper work done. The game does mention other bands in hail messages. 

Not sure if this is what you said, but we don't have the capability nor are we going to add playable videos themselves to the game, though.

>For missions, I was thinking of adding a few, and they would eventually join with the alien race eventually. There may be more missions made as I go on because I'm sort of a discovery writer.

>My second idea follows a writer who gets way too famous. You meet him when he is a beginner ideally at the beginning of your time as a pilot, and he'd be a friend for you throughout the game. You'd get missions from him every so often and he'd send you notes after the major points in the story. He gets along a lot and learns about each alien race sometime after you do. (You can tell him but if you don't he'll find out himself) At the end (Once the free world story is done) He gets to famous and attracts the attention of a powerful pirate lady who also has alien tech. I don't know how I'd be able to justify having this be a hard mission, but it is and at the end ship de-cloaks that is from an unknown alien race who help you fight of the pirates and then ask you for help.

>So this brings in the aliens, two alien races in a major plot. This might need to be done in sections later, namely after the republic side of the free world is done. I'll write that instead of this alien campaign if no one else is working on it because I'm currently stuck in my game because I don't want to join the free worlds and I think it'd better to get that done before going to the feature.

Again, we don't want any more aliens interaction with humans especially this deeply. We definitely have space for more races, though, should you want to consider adding them on their own in some manner, but it'd likely have to be one, not two. Also not sold on Pirates having alien weapons outside of in-lore Hai weapons.

>One thing that Endless sky doesn't have is a massive alien invasion story. A big campaign where aliens need to work together against a large alien attack. Two alien races that are so far out that they haven't been interfered with by Draks Quarg or Pug.(Somewhere Far, West and slightly north of the Deep.) One of these aliens are in what the quarg said is its expansion phase and have stumbled upon some weapons the Quarg warned you about on your first contact. I don't know exactly why they are but they are trying to invade south. I'll work that out.

The Pug invading FW space *is* a massive invasion story. If you're talking about something on a galactic scale, that isn't something that would fit appropriately as it would require both tons of lore connections between species, content creators of said species, etc.. And there is a lot of lore that suggests why this would never happen, and it's pretty implausible in the first place. There is a smaller idea that aliens invade the Core of the galaxy, but that's not going to be explored for a while until the Xapleaux are added (see below).

>The other alien race is basically another wish for endless sky. I found that the large battles are just you flying around and hoping your ships were stronger than the enemies. There may be a bit more skill to it, but I haven't learned it, nothing has really taught me in game. This other group of aliens would be weaker in raw strength but know such strategies. Things sort of like what TImothy Zahn has Grand Admiral Thrawn do in his books. Using cloaking devices to mimic shooting patterns, use fighters as projectiles etc. I don't think those things can be done in endless sky, but some strategies could be implemented, and I'm open to suggestions on what.

On continuous, we've actually just merged the Gegno, our first sub Tier 1 species, and there are plans to add more species weaker like this rather than just a bunch of higher ones. This will help fill that tier gap. As for strategies, the way you're suggesting they'd be is far too complex for anything Endless Sky would do. It could be told through missions, but as for static AI behavior, unlikely. As for these fancy cloak behaviors, fighters as projectiles, etc., these are all extremely ambitious and specific things that are probably unlikely in the current context of your desires.

>Basically at the beginning this other set of aliens (Which are militaristic) ask you for help. (Somehow the writer person has met them) They give you an advanced jump drive for the 15 some non hyperspace lane jumps you have to make. You do some missions but eventually the defending aliens would crumble, and the others would escape. I don't know what would happen next, but I'll figure something out.

We have another (maybe 2) possible future concept of the player helping a crumbling alien species, except without the militaristic part. Militaristic wars are pretty over-used at this point for plot points, so I strongly recommend avoiding them, at least on a simplistic level. We can still work with it and see where it goes should you chose to pursue an alien race, though.


On the Norislor:


>Norislor are a foot taller than you bulkier and have bluish gray skin. They wear armor. They are a war like species who conquers planets and fights for fun (Because there are species that are like that, Romans) But they are in human space for a reason.

We aren't adding any aliens into human space. As for "fights for fun", the Korath are somewhat like this, the Uhai do this, and the Gegno do it within duels with each other. I'm not so sure about "fights for fun, aka fights other species for fun to the death", though.

>They have planet destroying weapons and other terrible things. There tech level has to be a 4 so there on equal to Archon/ Drak or they'd get beaten up in a moment. They have jump drives.

We are not adding another Tier 4 race to the game. The Archons will always be the only ones around there. We are also likely not adding another Tier 3 race to the game, as there's already one more planned, and after that, we want to avoid having any more "fancy high tech races" as they're overdone at this point and don't help the game's overall story, and lore-wise, the Quarg would have to be as strong or much stronger than any problem issue species in the galaxy. We'd much prefer to add lesser tier species.

Also a no to having another (active)species with capable planet destruction weapons. The Korath had these, and were shortly shut down by Archons or else they would be destroyed. I doubt another species would ever be let into the galaxy, or exist in the galaxy, without being threated by the Archons.

>These guys have massive ships with massive outfit and weapons space so I think their outfit balance will err on the side of really high quality but really really big.

The largest ships in the game generally are left to the Korath, who treat all of their ships as their flying homes since they don't have any planets. The Korsmanath is probably the largest any one ship will get in the game, and generally we want to avoid anything else around that size. We aren't going to add anything more massive or larger than that, let alone a bunch more like it.

>If the player chooses to attack which is what the Quarg and maybe Archon want they gather forces from among the human and hai spaces (If that's what the Hai reveal outcome is) and then get slaughtered. It does halt the Norislor advance for some reason. You are out of options so the player goes see the language expert.

The Quarg wouldn't want any sort of war in the first place, moreso the Archons, who probably wouldn't get involved much at all. If the Quarg were to deem a new alien species dangerous and want people to team up against it, they would probably instead go take care of them themselves either with negotiations or threats before they become a problem. The whole "unite species" thing would be more of a Pug thing, and we wouldn't go as far as to have the Pug unite different species from across the galaxy.

>This may also require the Wander/ pug part of the story done because the next thing the player has to do is find a pug. The pug is no help they said they gave the crown to the people they got the weapons too. The Dver. Also they say that the Dver are quite close to earth. They are in a system that no one else uses.

Again, we aren't going to add another alien species into human space.

>The dver are basically dwarves from norse mythology and are a really old and advanced race without a planet. They live in a Dyson sphere cloaked around an uninhabited planet near earth. They know english. Also you can give the dvers certain peice of technology to "upgrade" if you give them raw materials (Astroids) It's a cool idea that encourages astroid mining and gives you upgrades.

We do have several advanced old races in game already, with a few more about to be added. We want to avoid this trope as it starts to mess with the lore of ancient species and the lack of a boom in life in the galaxy a long time ago. If we were to add a lot more, it wouldn't make sense. Ditto with the Dyson Sphere- we probably aren't going to add any fancy species that is capable of that. If anything, that'd be a Quarg thing.


I've been mentioning other species and plans as references to some of your points, so I'll list them here as a summary (keep in mind humans are Tier 1:


The Bunrodea are a Tier 1.5 species that will be added near the Core within the next patch, consisting of tech-savvy Praying Mantises that modify their bodies.

The Xapleaux are a Tier 1.7(?) species that will inhabit the Core of the galaxy and will be added in the next patch or two. They are caterpillar aliens that hide in the brightness of the core. A possible outsider invasion will take place with a future planned alien race. It won't be on a galactic scale, though.

The Gegno are a Tier 0.5-0.8 species that are already in vanilla in Continuous, the first aliens in the east side of the galaxy, and will be added next patch. They are Orcish humanoids that are at war with each other to figure out who is suitable to lead their race.  One faction loves war. The other loves technology.

The Rulei are an ancient extinct Tier 3 species that come with the Gegno. They previously inhabited Gegno space, and fell due to religious beliefs combined with deadly technology.

The Watchers are a Tier 3 species that live in the East that will be added in the next patch or two. They have strong cloaks and watch the Galaxy's events unfold from secret. 


There are plenty of other species and ideas that are under the process of being created and possibly considered for vanilla as well. I strongly recommend joining the Endless Sky discord if you haven't already, so you're able to get feedback from the community & creators as well as make sure you're not overlapping on concepts that others are already building around upon. We can also help you work around an alien species that could fit into vanilla's content more properly, because you do have some interesting ideas. If you want to continue pursuit of what you have laid out here though, by all means please make a plug-in, because I'm sure a lot of people would be interested in this content.

**Comment by TheShadowOfHassen** [comment]
So forget about the aliens. Apparently you have the aliens covered. I checked the wiki about contributing and while all your reasons make sense, none of them were in the wiki neither was a mention that you use discord.

So is just the human lore additions alright? I was planning on adding/ modifying some of the jobs to make them have a wider range of entertainment. Instead of just plays you could also rush delivery props for movies. Also add things like book transports or taking actors places.

I'd probably look over the human hails to see if they could be tweaked also in the same way also and add a few.

Then I wanted to add some missions/ culture conversations. One about helping to make a movie. One about taking someone on a book tour and one about taking an Orchestra on a first trip off planet. I'd add some culture conversations entertainment related.

I'm assuming the entertainment works the same way on planets as it does countries now. You can get famous on one planet but no one else could hear of you. The really famous however go to other planets. 

I can give outlines of the missions if you want.

The one idea that I had that could potentially be a no is something that gives Earth a better look than. The big bad city planet that everyone hates. It's sort of a weird ghost story that after you hear it and then help stranded ship get back to earth. Earth seems a lot better place. And you notice the green parts of the planet.
